src/topology/continuous_function/algebra.lean

```diff
@@ -639,6 +652,19 @@ instance has_scalar' {α : Type*} [topological_space α]
   has_scalar C(α, R) C(α, M) :=
 ⟨λ f g, ⟨λ x, (f x) • (g x), (continuous.smul f.2 g.2)⟩⟩
 
+instance {α : Type*} [topological_space α] [locally_compact_space α]
+  {R : Type*} [semiring R] [topological_space R]
+  {M : Type*} [topological_space M] [add_comm_monoid M]
+  [module R M] [has_continuous_smul R M] :
```

Comment:
I suspect you can relax this to
```suggestion
  {R : Type*} [topological_space R]
  {M : Type*} [topological_space M] [add_comm_monoid M]
  [has_scalar R M] [has_continuous_smul R M] :
```
but I haven't tried

Reply:
`[add_comm_monoid M]` can also be removed.

Reply:
Yes, it works without any of these as well.